.pre-commit-config.yaml

```diff
@@ -16,15 +16,6 @@ repos:
       - id: detect-aws-credentials
         args: [--allow-missing-credentials]
       - id: detect-private-key
-  # - repo: https://github.com/grantjenks/blue
-  #   rev: v0.9.1
-  #   hooks:
-  #     - id: blue
-  - repo: https://github.com/pycqa/flake8
-    rev: 6.0.0
-    hooks:
-      - id: flake8
-        exclude: "^(docs/experimental|tools)/"
```

Comment:
where is it reflected in ruff? 

ok to lint `docs` and `tools` but we should avoid to lint `experimental`

Reply:
Here https://github.com/fury-gl/fury/pull/881/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R103

Not sure why experimental should not be linted. The rest of the exclusions apply to the build, dist, etc. folders that are not kept under version control, but IMO anything under version control should be linted.

Reply:
experimental will be removed and we slowly do it step by step. It does not make any more sense to lint this folder. It contains deprecated code, bad formated code, etc. 
Everytime we move out something from experimental, we clean it to respect all the standard

Reply:
OK. Adding it to the excluded file list then.